plugins/woocommerce/src/Enums/OrderStatus.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Checkout\Enums;
+
+/**
+ * Enum class for all the order statuses.
+ *
+ * For a full documentation on the public order statuses, please refer to the following link:
+ * https://woocommerce.com/document/managing-orders/order-statuses/
+ */
+class OrderStatus
+{
```

Comment:
For some reason (we've also seen it on other PRs) the linter check isn't running properly. In any case, we should move the curly brace up:

```suggestion
class OrderStatus {
```

Relevant rule: `Generic.Classes.OpeningBraceSameLine.BraceOnNewLine`